db/pre-re/skill_db.yml

```diff
@@ -16322,6 +16322,68 @@ Body:
     MaxLevel: 1
     Flags:
       ShowScale: true
+  - Id: 721
+    Name: NPC_WIDEWEB
+    Description: Wide Web
+    MaxLevel: 1
```

Comment:
Might be best to just have this for each of these skills in pre-re?

Reply:
You mean, only Id, Name, Description and MaxLevel in pre-re?

Reply:
Right, since these skills and instance aren't part of pre-re.